docs/components/buttons/Example.vue

```diff
@@ -4,12 +4,12 @@
 
 <template>
     <div class="space-x-8">
-        <w-button primary>Click me</w-button>
-        <w-button primary loading>Click me</w-button>
-        <w-button>Click me</w-button>
-        <w-button secondary quiet>Click me</w-button>
-        <w-button secondary quiet small>Click me</w-button>
-        <w-button primary negative>Click me</w-button>
-        <w-button negative quiet>Click me</w-button>
+        <w-button primary>Primary</w-button>
+        <w-button primary loading>Loading</w-button>
+        <w-button utility>Utility</w-button>
+        <w-button>Secondary</w-button>
+        <w-button secondary quiet>Quiet</w-button>
+        <w-button primary negative>Negative</w-button>
+        <w-button negative quiet>Quiet</w-button>
```

Comment:
should we add "Secondary Quiet", "primary negative" and "negative quiet" in the button text as well to make it more clear?

Reply:
The "primary negative" example is a bit inconsistent across framework implementations because in Warp React it only requires `primary` prop to work. In Vue you need to set both primary and negative. In Warp Elements you'd set `variant="negative"` to get the same button 😅 I thought calling it just "Negative" here would be less error-prone for React and Elements users. Wdyt?

As for `negative/secondary quiet` I wanted the text to be shorter so buttons wouldn't wrap on desktop. But I can add that to be more clear and will fix the gaps when buttons wrap to the next line 😊

Reply:
Aha I see! In that case, yes that make sense :D